com.github.christianrauch.Jahresarbeit-2003.yaml

```diff
@@ -0,0 +1,121 @@
+app-id: com.github.christianrauch.Jahresarbeit-2003
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: /app/ja2003
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --device=dri
+cleanup:
+- /include
+- /bin
```

Comment:
```suggestion
```

You need the main program to be in there.

Reply:
As mentioned in the other commit, this will break the search path for resources (graphics, fonts, ...).